### ISSUE TITLE
Shaders are not initialized properly when creating ROs from scratch

### ISSUE THREAD

**Issue by louen** [AGGA-IRIT/Radium-Engine#115]
Ugly sphere !
![sphereglitch](https://cloud.githubusercontent.com/assets/1011507/17975019/7dca7b42-6ae8-11e6-90a3-9a54de95853c.png)

The color is incorrect and spheres not located at the origin have glitches. It appears that the objects do not use the proper BlinnPhong shader they should. (Maybe a problem in the default `ShaderConfig` ?

Selecting the BlinnPhong shader  in the UI makes the display correct again. After that, the RO cannot be put back in its original buggy state.

Full repro code :

```c++
class DummyComponent : public Ra::Engine::Component
{
public:
    DummyComponent() :  Ra::Engine::Component("comp") {}
    void initialize () override
    {
        std::shared_ptr<Ra::Engine::Mesh> displayMesh(new Ra::Engine::Mesh("sphere mesh"));
        displayMesh->loadGeometry(Ra::Core::MeshUtils::makeParametricSphere(0.1f));
        for (uint i = 0 ; i < 3 ; ++i)
        {
            std::string name  = "RO " + std::to_string(i);
            Ra::Engine::RenderObject* ro = Ra::Engine::RenderObject::createRenderObject(name,
                                                                                        this,
                                                                                        Ra::Engine::RenderObjectType::Fancy,
                                                                                        displayMesh);
            addRenderObject(ro);
            Ra::Core::Transform t;
            t.setIdentity();
            t.translate( Ra::Core::Vector3(i,0,0));
            ro->setLocalTransform(t);
        }
    }
};
```

 (add in e.g. `MainApplication::SetupScene()` )

```c++
 Ra::Engine::Entity* e =  Ra::Engine::RadiumEngine::getInstance()->getEntityManager()->createEntity("test");
 DummyComponent* c = new DummyComponent;
 e->addComponent(c);
 c->initialize();
```

**Comment by louen** [comment]
see commits e078b17 and 6660449.

**Comment by Zouch** [comment]
Fixed by commit  a086aac (and looks like I'm still not smart enough to close an issue from a commit ...)
